src/ad-helper.js

```diff
@@ -65,17 +65,17 @@ export function isAdPositionAllowed(el, win) {
 }
 
 /**
- * Returns the blessed container element, if the ad is contained by one.
+ * Returns the blessed container element tagName if the ad is contained by one.
  * This is called during layout measure.
  * @param {!Element} el
  * @param {!Window} win
- * @return {?Element}
+ * @return {?string}
  */
 export function getAdContainer(el) {
   do {
     el = el.parentNode;
     if (CONTAINERS[el.tagName]) {
-      return el;
+      return el.tagName;
     }
   } while (el && el.tagName != 'BODY');
   return null;
```

Comment:
I actually prefer `null` here. The caller should make sure there's no such implicit casting.